### PR TITLE
Add RFC README.md

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Description

Adds a draft README for the `rfcs` folder.

The format section is just based on my own past experience with this sort of thing and what I've seen working. I am eager for input here from others especially if there are missing sections that should be added.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [N/A] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [N/A] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by zackkrida** [comment]
I think a section about what the PR body should contain would be useful: eg,

- A summary of the proposed RFC
- A link to the full RFC (the GitHub link to the RFC file, most likely)

Alternatively @sarayourfriend we should probably make a new issue template for RFCs as part of this PR

**Comment by sarayourfriend** [comment]
@zackkrida It can't be an issue template because RFCs will be PRs. We could just include a comment in the PR template to add those sections when opening an RFC and delete the other ones?

**Comment by zackkrida** [comment]
> issue template because RFCs will be PRs

d'oh, of course 😊 

I guess I just wanted to avoid an empty PR body, and give folks a tldr before reading the markdown? But what you're suggesting is probably fine.

**Comment by zackkrida** [comment]
Some examples, just very minimal context and a link to the RFC: 

- https://github.com/reactjs/rfcs/pull/204
- https://github.com/reactjs/rfcs/pull/202

But I think you're right; it's not necessary 👍

**Review by zackkrida — CHANGES_REQUESTED** [review]
Some items we might want in the RFCs:

- Deadline for feedback
- A list of approvers

File: rfcs/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+3. Wait for feedback. Please `@` contributors you think might have specific and applicable knowledge to the problem space.
+4. Revise based on feedback.
+5. Continue until approval from two core contributors has been given and there are no absolute blockers raised.
+6. Create the related milestone and issues for the implementation plan. Link to the milestone in the final RFC and merge it.
+
+## Format
+
+There is no concrete format for RFCs but they probably (but not necessarily) should include the following sections:
+
+* The rationale/reason/goals for the proposed changes. Basically the high-level "why we need this".
```
Comment (sarayourfriend) [inline rfcs/README.md:25]:
```suggestion
* A list of approvers (to be filled in as approvals are given)
* A deadline for feedback (typically two weeks from the date the RFC is originally shared unless there are extenuating circumstances)
* The rationale/reason/goals for the proposed changes. Basically the high-level "why we need this".
```

**Review by zackkrida — APPROVED** [review]

**Review by obulat — APPROVED** [review]
Very nice and detailed README

**Pull request merged**